src/raw/mod.rs

```diff
@@ -0,0 +1,5 @@
+//! Raw poll APIs
+//!
+//! This module exposes raw Poll APIs.
+
+pub mod poll_evented;
```

Comment:
I think it would make sense to flatten the raw submodule.

```suggestion
mod poll_evented;
pub use poll_evented::*;
```